File: app/src/main/java/ir/touristland/Models/HotelDetails/RoomsItem.java
package ir.touristland.Models.HotelDetails;


import com.google.gson.annotations.SerializedName;

public class RoomsItem {

    @SerializedName("doubleBeds")
    private String doubleBeds;

    @SerializedName("image")
    private String image;

    @SerializedName("sofa")
    private String sofa;

    @SerializedName("reservationStatus")
    private boolean reservationStatus;

    @SerializedName("fullBoard")
    private boolean fullBoard;

    @SerializedName("description")
    private String description;

    @SerializedName("extraPersons")
    private int extraPersons;

    @SerializedName("sort")
    private String sort;

    @SerializedName("type")
    private String type;

    @SerializedName("isSpecialDate")
    private String isSpecialDate;

    @SerializedName("persons")
    private String persons;

    @SerializedName("isSpecial")
    private boolean isSpecial;

    @SerializedName("singleBeds")
    private String singleBeds;

    @SerializedName("nameFa")
    private String nameFa;

    @SerializedName("id")
    private int id;

    @SerializedName("beds")
    private String beds;

    @SerializedName("price")
    private String price;

    public String getDoubleBeds() {
        return doubleBeds;
    }

    public String getImage() {
        return image;
    }

    public String getSofa() {
        return sofa;
    }

    public boolean isReservationStatus() {
        return reservationStatus;
    }

    public void setReservationStatus(boolean reservationStatus) {
        this.reservationStatus = reservationStatus;
    }

    public boolean isFullBoard() {
        return fullBoard;
    }

    public String getDescription() {
        return description;
    }

    public int getExtraPersons() {
        return extraPersons;
    }

    public String getSort() {
        return sort;
    }

    public String getType() {
        return type;
    }

    public String getIsSpecialDate() {
        return isSpecialDate;
    }

    public String getPersons() {
        return persons;
    }

    public boolean isIsSpecial() {
        return isSpecial;
    }

    public String getSingleBeds() {
        return singleBeds;
    }

    public String getNameFa() {
        return nameFa;
    }

    public int getId() {
        return id;
    }

    public String getBeds() {
        return beds;
    }

    public String getPrice() {
        return price;
    }

    public void setPrice(String price) {
        this.price = price;
    }
}

File: app/src/main/java/ir/touristland/Models/HotelDetails/RoomsItem.kt
package ir.touristland.Models.HotelDetails


import com.google.gson.annotations.SerializedName

class RoomsItem {

    @SerializedName("doubleBeds")
    val doubleBeds: String? = null

    @SerializedName("image")
    val image: String? = null

    @SerializedName("sofa")
    val sofa: String? = null

    @SerializedName("reservationStatus")
    var isReservationStatus: Boolean = false

    @SerializedName("fullBoard")
    val isFullBoard: Boolean = false

    @SerializedName("description")
    val description: String? = null

    @SerializedName("extraPersons")
    val extraPersons: Int = 0

    @SerializedName("sort")
    val sort: String? = null

    @SerializedName("type")
    val type: String? = null

    @SerializedName("isSpecialDate")
    val isSpecialDate: String? = null

    @SerializedName("persons")
    val persons: String? = null

    @SerializedName("isSpecial")
    val isIsSpecial: Boolean = false

    @SerializedName("singleBeds")
    val singleBeds: String? = null

    @SerializedName("nameFa")
    val nameFa: String? = null

    @SerializedName("id")
    val id: Int = 0

    @SerializedName("beds")
    val beds: String? = null

    @SerializedName("price")
    var price: String? = null
}